### DISCUSSION TITLE
Invitation code for registration

### DISCUSSION THREAD

**Discussion by danny2jenny** [filebrowser/filebrowser#4924] [Ideas]
It‘s possible to add a invitation code for registration? 

Adminstrator can set a invitation code in Global Settings, then only people have this code can regist successfully.

This feature is quite useful for registration.

Thanks!

**Comment by o1egl** [comment]
> It‘s possible to add a invitation code for registration?
No filebrowser has no invitation mechanism

**Comment by jniggemann** [comment]
Thank you for opening an issue and contributing a feature request to the filebrowser project!
While we're looking for maintainers, we limited the issue tracker to actual bugs. Feature requests are now tracked in the  [discussions](https://github.com/filebrowser/filebrowser/discussions).
Your idea is valuable for us and as such, we're moving it there.


> [!WARNING]  
> This project is currently on maintenance-only mode, and is looking for new maintainers. For more information, please read the discussion #4906. Therefore, please note the following:
>
>- It can take a while until someone gets back to you. Please be patient.
>- Issues are only being used to track bugs. Any unrelated issues will be converted into a discussion.
>- No new features will be implemented until further notice. The priority is on triaging issues and merge bug fixes.
>
>If you're interested in maintaining this project, please reach out via the discussion above.
